ftcosi/protocol/struct.go

```diff
@@ -76,7 +76,12 @@ type StructCommitment struct {
 
 // Challenge is the ftcosi challenge message
 type Challenge struct {
+	// CoSiChallenge is deprecated and should not be used anymore!
```

Comment:
This does not correspond to the code, i.e. sup_protocol.go, line 280 where you compare the locally calculated challenge with the remotely provided one. If it should not be used anymore, why are we depending on it being set in order to check it?